requirements.txt

```diff
@@ -1,11 +1,7 @@
 praw==4.4.0
 redis<3.0.0
-hiredis
-better_exceptions
 addict
-cython
```

Comment:
Fair point. I actually read up on Cython and how to use it. Didn't see it used in this project so I figured it was a nested dependency.

I suppose it should be documented why this is being manually added. I'll hop on that and readd this dependency.